libxcvt.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: libxcvt
+  version: 0.1.2
+  epoch: 0
+  description: "VESA CVT standard timing modeline generation library & utility"
+  copyright:
+    - license: custom
```

Comment:
We can't accept packages that aren't licensed under accepted licenses. Can you give more details about the license for this package?

Reply:
sorry, i forgot include it. 

i've added [licence](https://gitlab.freedesktop.org/xorg/lib/libxcvt/-/blob/master/COPYING)